src/app/mail/mail-settings/addresses-signature/addresses-signature.component.html

```diff
@@ -1,7 +1,15 @@
 <!-- Signature Section -->
-<section class="mailbox-section">
+<section id="signature" class="mailbox-section">
   <header class="ui-header ui-header-bordered">
     <h5 class="ui-header-subtitle text-dark mb-0">
+      <a class="anchor" aria-hidden="true" (click)="onAnchoredLink('signature')">
+        <svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true">
```

Comment:
@superdev0714 
Inline svg aren't cached. Please use a proper method, we have many icons libraries and now we add icons like this...

Reply:
@superdev0714